components/front/first.tsx

```diff
@@ -141,8 +141,33 @@ const First = () => {
 						styles.hackscBackground
 					)}
 				>
-					<span className={styles.hacksc}>
-						HackSC HackSC HackSC HackSC HackSC HackSC HackSC
+					<span className={styles.hacksc}>Hack</span>
```

Comment:
for these, you can use javascript's array.map() function so you don't need to list them all individually:
`{ new Array(7).map((index) => {
  return (
  div key={index.toString()}
    span className={styles.hacksc}>Hack</span>
    span className={styles.sc}>
      b>SC </b>
    /span>
  /div>
)} )}`

Reply:
had to remove front `<` character because they were rendered as actual code by markdown :/